examples/install_simple/main.tf

```diff
@@ -14,9 +14,30 @@
  * limitations under the License.
  */
 
+provider "google" {
+  version = "~> 2.11.0"
+  project = var.project_id
+}
+
+provider "local" {
+  version = "~> 1.3"
+}
+
+provider "null" {
+  version = "~> 2.0"
+}
+
+provider "template" {
+  version = "~> 2.0"
+}
+
+provider "random" {
+  version = "~> 2.0"
+}
+
 module "forseti-install-simple" {
   source  = "terraform-google-modules/forseti/google"
-  version = "~> 5.0.0"
+  version = "~> 5.1.1"
```

Comment:
This should remain as 5.0.0. 5.1.1 does not exist yet and that's why the lint test failed. The pessimistic operator will use >= 5.0.0 and < 6.0.0, so `~> 5.0.0` will use 5.1.1 once it's released.